src/commands/secret/mod.rs

```diff
@@ -35,25 +36,50 @@ fn validate_target(target: &Target) -> Result<(), failure::Error> {
 fn secret_errors(error_code: u16) -> &'static str {
     match error_code {
         7003 | 7000 => {
-            "Your wrangler.toml is likely missing the field \"account_id\", which is required to write to Workers KV."
+            "Your wrangler.toml is likely missing the field \"account_id\", which is required to create a secret."
         }
```

Comment:
can we add a message for delete/list 10007 errors here?

Reply:
We'll handle this in a separate ticket given that not found errors for both workers and bindings are both error code 10007.